### PR TITLE
[TCA-41]: Add Error Page

### PR DESCRIPTION
**Pull request by CodeLeom**
Please review sir @0xJamin

### CODE REVIEW

File: app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <Image
+          src={background}
+          alt="background grid for the page body"
+          className="w-full"
+          width={1000}
+          height={1000}
+        />
+      </div>
+
+      <div className="w-screen h-screen flex justify-center items-center">
```
Comment (codejagaban) [inline app/not-found.tsx]: use `container` here

File: app/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import Image from "next/image";
+import background from ".././public/body-bg.svg";
+
+export default function NotFound() {
+  return (
+    <div>
```
Comment (codejagaban) [inline app/not-found.tsx]: This should be a `section`

File: app/not-found.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          className="w-full"
+          width={1000}
+          height={1000}
+        />
+      </div>
+
+      <div className="w-screen h-screen flex justify-center items-center">
+        <div className="w-9/12 h-3/6 flex justify-center items-center p-6">
+          <div>
+            <h1 className="text-6xl font-bold text-red-600">Oops</h1>
```
Comment (codejagaban) [inline app/not-found.tsx]: Update heading to match design

File: app/not-found.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            <p className="text-4xl font-semi-bold text-gray-500">
```
Comment (codejagaban) [inline app/not-found.tsx]: update font size

File: app/not-found.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+              Something went wrong
+            </p>
+            <p className="text-xl font-normal text-gray-500 pb-14">
+              You landed on the wrong page. Please check the link again, go back
+              Home or reach out to us
+            </p>
+
+            <div className="gap-8">
+              <button className="bg-black text-white font-bold py-5 px-10 rounded inline-flex tracking-wider mr-8">
```
Comment (codejagaban) [inline app/not-found.tsx]: Buttons should be reusable components, and should fit design

File: app/not-found.tsx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+                    stroke-linejoin="round"
+                  />
+                </svg>
+                <span className="ml-2">Contact Support</span>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div>
+          <Image
```
Comment (codejagaban) [inline app/not-found.tsx]: Image size should match design

**Review by codejagaban — CHANGES_REQUESTED** [review]
@CodeLeom  some changes to update here Boss

**Review by codejagaban — CHANGES_REQUESTED** [review]
@CodeLeom  I'm creating a migration from Nextjs to [Astro](https://astro.build/),

So unfortunately you'd have to rework on this PR again after pulling the changes from the `develop` branch (please hold on the migration is not done yet)

We'll still be using `react` components where necessary, but Astro is by default, our choice

**Comment by codejagaban** [comment]
You can move your files to the `old` folder and when you've replicated the changes in Astro, we can then delete it

**Comment by codejagaban** [comment]
@CodeLeom  Please convert this component to an Astro component.
Thanks

**Comment by CodeLeom** [comment]
made a new pr

**Pull request closed without merging**